src/main/java/OKRReader.java

```diff
@@ -50,23 +50,19 @@ public static void readOKR() {
         Scanner scanner = new Scanner(System.in);
 
         try (Reader reader = Files.newBufferedReader(Paths.get("C:\\Users\\mercedes.tarrio\\Downloads\\OKR.csv"))) {
-            // columns name
             String[] columns = {"objective", "keyResult1", "keyResult2", "keyResult3", "keyResult4", "keyResult5"};
 
-            // create a mapping strategy
             ColumnPositionMappingStrategy strategy = new ColumnPositionMappingStrategy();
             strategy.setType(CSVEntry.class);
             strategy.setColumnMapping(columns);
 
-            // create csv bean reader
             CsvToBean csvToBean = new CsvToBeanBuilder(reader)
                     .withMappingStrategy(strategy)
                     .withSkipLines(1)
                     .withIgnoreLeadingWhiteSpace(true)
                     .build();
 
-            // iterate through okr
-            for (CSVEntry okr : (Iterable<CSVEntry>) csvToBean) { // transform here
+            for (CSVEntry okr : (Iterable<CSVEntry>) csvToBean) {
```

Comment:
why do we need to do a parsing in a for loop? What is the responsibility of the `CsvToBean` class? If it is simply parsing the CSV file, could it not simply return an iterable in a method rather than us parsing it?